app/graphql/irida_schema.rb

```diff
@@ -9,7 +9,7 @@ class IridaSchema < GraphQL::Schema # rubocop:disable GraphQL/ObjectDescription
   use GraphQL::Dataloader
 
   max_depth 15
-  max_complexity 250
+  max_complexity 500
```

Comment:
**nitpick (blocking):** Can you please remove this change, for queries that you get complexity warnings, usually listing samples etc. you can resolve that by adding `first: 10`. We will likely need to adjust this at some point, but for now please leave it as is.

Reply:
Fixed.